docs/API-functions.md

```diff
@@ -968,7 +968,7 @@ connection parties.
 
 #### SRT_REJ_FILTER
 
-The `SRTO_FILTER` option has been set differently on both connection
+The `SRTO_PACKETFILTER` option has been set differently on both connection
```

Comment:
Note: this was added to the dox earlier while adding the "reject reason" feature and this flag was at that time known under this name, although it was only about to be added in THIS feature.